src/js/_enqueues/vendor/thickbox/thickbox.js

```diff
@@ -76,15 +76,16 @@ function tb_show(caption, url, imageGroup) {//function called when the user clic
 	   		baseURL = url;
 	   }
 
-	   var urlString = /\.jpg$|\.jpeg$|\.png$|\.gif$|\.bmp$|\.webp$/;
+	   var urlString = /\.jpg$|\.jpeg$|\.png$|\.gif$|\.bmp$|\.webp$|\.avif$/;
```

Comment:
> I think we would need to consider `.heif` files separately

Is there a user need for that?

> are they technically AVIF files?

According to the specification I would say yes, without certitude. But I doubt any library generates them this way.  
It would be `heif` as a major_brand and `avif` as a compatible_brand.  
Anyway this is not even worth a discussion to be honest.

> so far only Safari support heif

I think you meant HEIC. I agree it is a bit confusing and finding HEIF associated with HEIC everywhere does not help.  
Also there is little interest for browsers to support the generic container HEIF without the specific underlying formats (AV1, HEVC). That is why there is AVIF or HEIC support but it won't mention MIAF, HEIF or ISOBMFF support because these pure containers cannot be rendered on browsers.  
CMS like WordPress on the other end could take advantage of the codec-agnostic image properties such as width/height defined directly in HEIF property `ispe` (as long as it is known that there is enough browser support for it). I assume WordPress does not care about pixel values but as you mentioned, it might handle resizing and stuff so my point is only for page layout purposes.

Reply:
> I think you meant HEIC. I agree it is a bit confusing and finding HEIF associated with HEIC everywhere does not help.

Ah right, I was confusing the two.